src/util/port_util.js

```diff
@@ -35,7 +35,13 @@ const util = {
     let attempts = 0;
 
     const acquire = () => {
-      checkPorts([util.getNextPort()], (result) => {
+      // check all the ports in the range. If fail, move to next set
+      const startPort = util.getNextPort();
+      const ports = [];
+      for(let i=startPort;i<startPort + settings.BASE_PORT_SPACING; i++){
```

Comment:
shouldn't you be using here `settings. BASE_PORT_RANGE ` instead of `settings.BASE_PORT_SPACING`? 

Also optionally incrementing `i` by `settings.BASE_PORT_SPACING` (assuming that's what the spacing if for)?

something like:
```
for(let i = startPort; i < startPort + settings.BASE_PORT_RANGE; i += BASE_PORT_SPACING){
```

Reply:
No, my comments were wrong. I corrected them. This PR is to check all ports in the spacing so that we don't just check a port, and then assume the next consecutive ports are free.

Reply:
👍